backend/prompt_studio/prompt_studio_registry/prompt_studio_registry_helper.py

```diff
@@ -119,25 +119,14 @@ def get_tool_by_prompt_registry_id(
                 f"ID {prompt_registry_id}: {e} "
             )
             return None
-        # The below properties are introduced after 0.20.0
-        # So defaulting to 0.20.0 if the properties are not found
-        image_url = prompt_registry_tool.tool_metadata.get(
-            JsonSchemaKey.IMAGE_URL, "docker:unstract/tool-structure:0.0.20"
-        )
-        image_name = prompt_registry_tool.tool_metadata.get(
-            JsonSchemaKey.IMAGE_NAME, "unstract/tool-structure"
-        )
-        image_tag = prompt_registry_tool.tool_metadata.get(
-            JsonSchemaKey.IMAGE_TAG, "0.0.20"
-        )
         return Tool(
             tool_uid=prompt_registry_tool.prompt_registry_id,
             properties=Properties.from_dict(prompt_registry_tool.tool_property),
             spec=Spec.from_dict(prompt_registry_tool.tool_spec),
             icon=prompt_registry_tool.icon,
-            image_url=image_url,
-            image_name=image_name,
-            image_tag=image_tag,
+            image_url=settings.STRUCTURE_TOOL_IMAGE_URL,
```

Comment:
@Deepak-Kesavan QQ : Will his throw an error f the config variable is missing?

Reply:
It can't be missing. It is the env we set in .env of the backend.

Reply:
I mean if the variable is not set in the env will it throw an error?

Reply:
I mean if the variable is not set in the env will it throw an error?

Reply:
The backend won't even boot up if it was not there. Check this for additional information.

https://github.com/Zipstack/unstract/blob/7c513ff86b2a7b28f06fafbdd1760b77d4003594/backend/backend/settings/base.py#L148-L150C68